### PR TITLE
Code Samples 

### PR DESCRIPTION
**Pull request by suryap649**
Added section to show case code samples

### CODE REVIEW

File: docs/source/software/softwareoverview.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ Then, you must set the installation path to a directory where you have write acc
 You can use `./configure` to do this, such as  `./configure --prefix=/home/<yourusername>/software`
 After you have set the installation path, you need to compile the code using `make` and then install the software using `make install`.
 
+## Getting Started Scientific Software Applications
+
+The northeastern-rc team has created a collection of [scripts](https://github.com/northeastern-rc/discovery-example-scripts) to assist you in getting started with different scientific software packages on discovery. Although these scripts may not have all the solutions, they aim to provide users with a starting point to grasp the syntax and procedures used in various applications.
```
Comment (joseph-cooke) [inline docs/source/software/softwareoverview.md:37]: northeastern-rc -> Research Computing
discovery -> the HPC

File: docs/source/software/softwareoverview.md [hunk 2]
```diff
@@ -32,6 +32,16 @@ Then, you must set the installation path to a directory where you have write acc
...
+
+- [Matlab](https://github.com/northeastern-rc/discovery-example-scripts/tree/main/MATLAB)
+- [Multiprocessing](https://github.com/northeastern-rc/discovery-example-scripts/tree/main/Multiprocessing)
+- [Schrodinger](https://github.com/northeastern-rc/discovery-example-scripts/tree/main/Schrodinger)
+
+The collection of scripts is continuously growing, and the team welcomes any contributions to the [project](https://github.com/northeastern-rc/discovery-example-scripts).
```
Comment (joseph-cooke) [inline docs/source/software/softwareoverview.md:43]: growing, -> growing

**Comment by joseph-cooke** [comment]
Check with Manasvita if we want to advertise for contributions to the github for this project but other than that, just a couple small changes. Thank you!!

**Review by manasvita — APPROVED** [review]
Looks good.

**Comment by manasvita** [comment]
> Check with Manasvita if we want to advertise for contributions to the github for this project but other than that, just a couple small changes. Thank you!!

@joseph-cooke, did you mean having user-contributed scripts section/repo on our Github?

**Comment by joseph-cooke** [comment]
> > Check with Manasvita if we want to advertise for contributions to the github for this project but other than that, just a couple small changes. Thank you!!
> 
> @joseph-cooke, did you mean having user-contributed scripts section/repo on our Github?

@manasvita, yes, I wasn't sure if we were advertising for users to submit scripts for us to host in this repo for the scripts and examples or not. Thank you!!

**Comment by manasvita** [comment]
> > > Check with Manasvita if we want to advertise for contributions to the github for this project but other than that, just a couple small changes. Thank you!!
> > 
> > 
> > @joseph-cooke, did you mean having user-contributed scripts section/repo on our Github?
> 
> @manasvita, yes, I wasn't sure if we were advertising for users to submit scripts for us to host in this repo for the scripts and examples or not. Thank you!!

@joseph-cooke, ok, got it. We haven't done that yet but should certainly do it in the next few iteration.

**Pull request merged**